config/labels/analyzers/clangsa.json

```diff
@@ -326,6 +327,13 @@
       "doc_url:https://clang.llvm.org/docs/analyzer/checkers.html#alpha-webkit-uncountedlocalvarschecker",
       "profile:extreme"
     ],
+    "apiModeling.Errno": [
+      "profile:default",
+      "profile:extreme",
+      "profile:portability",
+      "profile:sensitive",
+      "severity:LOW"
+    ],
```

Comment:
We should never explicitly mention apiModeling checkers, it might lead to them being disabled!